### ISSUE TITLE
[PM-623] Planner does not check validity of src in nonsharedfs mode

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#741]
I tried to run a workflow in nonsharedfs mode, and it was not until job started failing I realized that I had used file:// for the staging site (local site in this case). In the PegasusLite scripts I saw transfers like:

# src 1 local
file:////local-scratch/rynge/wf/scratch/rynge/pegasus/Run1/20120620T143528+0000/Run1-25_ID0000025/periodogram
# dst 1 condorpool
file://$PWD/periodogram

The src  would not be valid on the remote side.

I think the planner should do some pre-flight checks on the validity of the URLs it puts in the PegasusLite scripts.

**Reporter:** @rynge
**Resolution:** Won't Fix
**Watchers:**
@vahi

**Comment by mayani** [comment]
**Author:** @vahi

sometimes users have logical sites in site catalog that map to same underlying physical site
